src/components/app.tsx

```diff
@@ -16,7 +16,14 @@ export class AppComponent extends BaseComponent<IProps, IState> {
     const {ui} = this.stores;
     return (
       <div className="app">
-        <Text text={ui.sampleText} />
+        <MagnetCanvas
+          width={800}
+          height={500}
+          showMagnet1={true}
+          showMagnet2={true}
+          showGradient={true}
+          showVectors={true}
+        />
```

Comment:
I'll move this into main-content component when I merge.

Reply:
:+1: